app/controllers/application_controller.rb

```diff
@@ -71,4 +73,9 @@ def locale_from_header
       lang_header.scan(/^[a-z]{2}/).first.to_sym
     end
   end
+
+  def set_raven_context
+    Raven.user_context(id: current_user.id, email: current_user.email) # or anything else in session
```

Comment:
adding `if current_user`to the end of line 78 fixed specs. Don't love changing application code to fix specs, so feel free to do something else.